CHANGELOG.md

```diff
@@ -58,8 +58,6 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Use native `pipe2` on all BSD targets.  Users should notice no difference.
   ([#777](https://github.com/nix-rust/nix/pull/777))
-- `AioCb::from_boxed_slice` no longer allows `aio_read`
```

Comment:
Is there a reason why we have this step fixing up `from_boxed_slice` only to remove it in the following commit?

Reply:
Not really.  I should squash them together.